cdap-ui/graphql/resolvers/scheduleResolvers.js

```diff
@@ -23,30 +23,14 @@ cdapConfigurator.getCDAPConfig().then(function(value) {
   cdapConfig = value;
 });
 
-const runsResolver = {
-  Workflow: {
-    runs: async (parent, args, context) => {
-      const namespace = context.namespace;
-      const name = parent.app;
-      const workflow = parent.name;
-      const options = resolversCommon.getGETRequestOptions();
-      options.url = urlHelper.constructUrl(
-        cdapConfig,
-        `/v3/namespaces/${namespace}/apps/${name}/workflows/${workflow}/runs`
-      );
-
-      return await resolversCommon.requestPromiseWrapper(options);
-    },
-  },
-};
-
 const schedulesResolver = {
   Workflow: {
     schedules: async (parent, args, context) => {
       const namespace = context.namespace;
       const name = parent.app;
       const workflow = parent.name;
       const options = resolversCommon.getGETRequestOptions();
+      resolversCommon.addAuthorizationTokenIfPresent(context.auth, options);
```

Comment:
since you are adding this in every single resolvers, and every single resolvers is actually calling the `requestPromiseWrapper`, I think you should add the Authentication token there to keep it in one place

Reply:
Done